lib/server.go

```diff
@@ -2161,20 +2166,33 @@ func (srv *Server) StartStatsdReporter() {
 	}()
 }
 
-func (srv *Server) _handleAddrMessage(pp *Peer, msg *MsgDeSoAddr) {
+func (srv *Server) _handleAddrMessage(origin *Peer, desoMsg DeSoMessage) {
```

Comment:
nit: pp is the common and accepted variable name for the Peer type in incoming messages handlers. Revert this for consistency with the rest of the code
```suggestion
func (srv *Server) _handleAddrMessage(pp *Peer, desoMsg DeSoMessage) {
```